src/concrete/ml/quantization/base_quantized_op.py

```diff
@@ -257,6 +257,9 @@ def dump_dict(self) -> Dict:
             if attribute_name not in metadata:
                 metadata[attribute_name] = attribute_value
 
+        # Sort the metadata to be sure it is always saved in the same order
+        metadata = dict(sorted(metadata.items()))
```

Comment:
worth doing the same for built-in models if you think there's a real point of having that .. but it's going to be many copy-paste

Reply:
Why not done with an `OrderedDict` if the order is so important?

Reply:
I don't think it's important, but I guess it was for debugging purposes

Reply:
It's important for our tests where we check that multiple serialization are the same.